code/modules/paperwork/paper.dm

```diff
@@ -345,6 +361,33 @@
 			else
 				to_chat(user, SPAN_DANGER("You must hold \the [P] steady to burn \the [src]."))
 
+/obj/item/paper/verb/seal_paper()
+	set name = "Seal paper"
+	set category = "Object"
+	set src in usr
+
+	var/confirm = alert(usr, "Are you sure you wish to seal this document?\n\nNote: This will prevent the edit of any fields on the paper,\nexcluding the end of the page.", "Confirm", "Yes", "No")
+	if(confirm != "Yes")
+		return FALSE
+	info = replacetext(info, "<span class=\"paper_field\"></span>", "<br>")
```

Comment:
This will only eliminate unwritten fields (because there'll be content between the <span>...</span> tags. You need different handling to seal written fields. 
![image](https://github.com/user-attachments/assets/4f2f3b1f-fdff-4d1f-97f8-546339a08b31)